App/Controllers/Admin/Home.php

```diff
@@ -3,24 +3,227 @@
 namespace App\Controllers\Admin;
 
 
+use App\Models\RoleModel;
+use App\Models\UserModel;
+use Core\Constant;
+use Core\Container;
+use Core\Traits\PasswordFunctions;
+use Core\Traits\StringFunctions;
 
 class Home extends \Core\AdminController
 {
+    use StringFunctions;
+    use PasswordFunctions;
+    protected $siteConfig;
+    protected $pagination;
+
+    private $userModel;
+    private $roleModel;
+
+    public function __construct(Container $container)
+    {
+        $this->loadModules[] = 'SiteConfig';
+        $this->loadModules[] = 'pagination';
+        parent::__construct($container);
+        $this->userModel = new UserModel($this->container);
+        $this->roleModel = new RoleModel($this->container);
+
+        $this->data['configs'] = $this->siteConfig->getSiteConfig();
+    }
+
+    /**
+     * The front page of the admin section. We display the user info
+     * @throws \ReflectionException
+     * @throws \Twig_Error_Loader
+     * @throws \Twig_Error_Runtime
+     * @throws \Twig_Error_Syntax
+     */
     public function index()
     {
-        //testing auth
-        if ($this->auth->isAdmin()) {
-            $this->data['userRole'] = 'Admin';
-            $this->data['userLevel'] = $this->auth->getUserLevel();
-        } elseif ($this->auth->isUser()) {
-            $this->data['userRole'] = 'User';
```

Comment:
use a constant for 'User'

Reply:
Resolved in branch Security_and_bug_fixes